Tests/Editor/Actions/StripProBuilderScripts.cs

```diff
@@ -0,0 +1,48 @@
+﻿using UObject = UnityEngine.Object;
+using System.Collections.Generic;
+using NUnit.Framework;
+using UnityEditor.ProBuilder;
+using UnityEngine;
+using UnityEngine.ProBuilder;
+using UnityEditor;
+
+public class StripProBuilderScripts
+{
+    bool m_OpenedWindow = false;
+    ProBuilderMesh m_cube;
+    GameObject m_cube_obj;
```

Comment:
In general I'd suggest declaring variables in the smallest possible scope. Since these are only used in a single function, there's no need to declare them at a class level. If for example you were to reuse these in multiple functions in this class, possibly with a `[SetUp]` and `[TearDown]` then it would make sense to keep them at this scope.